R/task.R

```diff
@@ -679,11 +679,17 @@ print_info_data <- function(data) {
   }
   cli::cli_li("Environment: {data$environment}")
   if (NROW(data$envvars) > 0) {
-    cli::cli_li("Environment variables: {data$envvars$name}")
+    cli::cli_li("Environment variables:")
+    env_vars_ul <- cli::cli_ul()
```

Comment:
https://cli.r-lib.org/reference/cli_dl.html perhaps, possibly as

```
cli::cli_dl(items = set_names(data$envvars$value, data$envvars$name)
```

For secrets though, you should only print stars!